src/main/java/com/google/graphgeckos/dashboard/storage/DataRepository.java

```diff
@@ -17,6 +17,7 @@
 import com.google.graphgeckos.dashboard.datatypes.BuildBotData;
 import com.google.graphgeckos.dashboard.datatypes.BuildInfo;
 import com.google.graphgeckos.dashboard.datatypes.GitHubData;
+import java.rmi.NotBoundException;
```

Comment:
It's a bad practice to repurpose exceptions from packages that you are not using. The `NotBoundException` has specific semantics with regard to RMI. Instead, consider declaring your own exception types and document their semantics.

Reply:
Addressed in #170 . Thank you!